com.woltlab.wcf/templates/unfurlUrl.tpl

```diff
@@ -0,0 +1,14 @@
+{if $object->status == "SUCCESSFUL"}
+	<div class="unfurlCard {if $object->imageType == 'COVER' && !$object->getImageUrl()|empty}unfurlLargeContentImage{elseif $object->imageType == 'SQUARED' && !$object->getImageUrl()|empty}unfurlSquaredContentImage{/if}">
+		<a href="{$object->url}"{if EXTERNAL_LINK_REL_NOFOLLOW || EXTERNAL_LINK_TARGET_BLANK} rel="{if EXTERNAL_LINK_REL_NOFOLLOW}nofollow{/if}{if EXTERNAL_LINK_REL_NOFOLLOW && EXTERNAL_LINK_TARGET_BLANK} {/if}{if EXTERNAL_LINK_TARGET_BLANK}noopener noreferrer{/if}"{/if}{if EXTERNAL_LINK_TARGET_BLANK} target="_blank"{/if}>
+			<div{if !$object->getImageUrl()|empty} style="background-image: url('{$object->getImageUrl()}')"{/if}></div>
```

Comment:
Will this cause issues if the image URL contains a `'`?

Reply:
Correct me, if I am wrong, but URLs, which contains `'` are not valid. I can't find any valid URL which contains a `'`. 

Invalid URLs are not stored and are rejected (See https://github.com/WoltLab/WCF/pull/4036/files#diff-a83e91d395d63d2a6354a05b163ddce84bb12f5536eacdca1393cc8113538f87R90).